Documentation/IntroductionPackage/Index.rst

```diff
@@ -51,7 +51,19 @@ To install the Introduction Package run the following command:
 
          ddev composer require typo3/cms-introduction
 
-This command will download and activate the extension.
+Since TYPO3 11.5 it is essential to setup the extension afterwards by running
+the following command:
+
+.. tabs::
+
+   .. group-tab:: bash
+
+       .. code-block:: bash
+
+         composer exec typo3 extension:setup
```

Comment:
We use in the docs `vendor/bin/typo3 extension:setup` and `typo3/sysext/core/bin/typo3 extension:setup`. For consistency, this should be used also here.

Have a look: https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/CommandControllers/Index.html#run-a-command-from-the-command-line

Also take care with the indentations. We use 4 chars.

Reply:
We will do that in a follow up